R/cnd.R

```diff
@@ -378,7 +379,7 @@ interrupt <- function() {
 #' If you call `cnd_muffle()` with a condition that is not mufflable
 #' you will cause a new error to be signalled.
 #'
-#' * Errors are not mufflable since they are signalled in critical
```

Comment:
After reading https://dictionary.cambridge.org/grammar/british-grammar/as-because-or-since it is not clear to me these changes are desired.

Reply:
Agreed. In a few cases (such as this one), I think adding a comma after the since- clause helps emphasize that.

Reply:
I don't like this comma :(

Reply:
While the top one makes sense to me, the bottom comma (though technically correct/stylistic) seems to break things up awkwardly, given that it's a bullet.

Reply:
This comma is no more. I also rebased, to include the typo fix from #544. Please let me know if there's anything else you'd like changed.

Reply:
Fewer commas throughout please. Only add a comma if you're absolutely sure it is an improvement that everyone will agree with. In general this advice should hold for all copy-editing changes, i.e. things that are not typos, clear grammatical errors, or that do not clearly improve upon the clarity of the doc.

Also in the future it is better to create multiple PRs that contain only similar changes. This way we can move forward faster. Thanks for bearing with us!

Reply:
Thanks for the quick reply. Sounds good; I'll review and revert the comma changes. Quick question about multiple PRs: Should they build on each other (i.e., each as a separate branch that builds on the last), or each a new branch based on master? Thanks!

Reply:
If they are independent it is better to base them on master. If they are dependent it is ok to have them based on one another.